### PR TITLE
Conda-Forge Recipe for Plottr

### PR DESCRIPTION
**Pull request by trevormorgan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jenshnielsen** [comment]
@trevormorgan I think you will need to move the recipe from the example folder to a plottr folder

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -17,66 +15,55 @@ source:
...
   # List all the packages/modules that `run_test.py` imports.
   imports:
-    - simplejson
-    - simplejson.tests
+    - qcodes
+    - pytest
+    - pytest-qt
+    - mypy
+    - PyQt5-stubs
+    - git+https://github.com/numpy/numpy-stubs.git
```
Comment (jenshnielsen) [inline recipes/example/meta.yaml:51]: I think this should just be `- plottr` which will basically test that `plottr` can be opened when the conda package is installed

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,10 +1,8 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
```
Comment (jenshnielsen) [inline recipes/example/meta.yaml:3]: I think you are intended to remove most of the comments

File: recipes/example/meta.yaml [hunk 3]
```diff
@@ -17,66 +15,55 @@ source:
...
   # You may need the openssl package, available on conda-forge:
   #  `conda install openssl -c conda-forge``
 
 build:
-  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
-  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
-  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
-  # noarch: python
+  noarch: python
   number: 0
```
Comment (jenshnielsen) [inline recipes/example/meta.yaml:26]: I think we need to skip python 3.6 and 2.7 here.

```
  skip: true  # [py27]
  skip: true  # [py36]
```

**Comment by jenshnielsen** [comment]
@trevormorgan  I think you might need to follow the advice here https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib to add a yum_requirements.txt file

File: recipes/plottr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    A framework for inspecting data, based on flowcharts from pyqtgraph.
+    plottr's main aim is to allow the user to define custom data processing flows and plotting.
+    A particular use case is data filtering and plotting.
+  doc_url: https://plottr.readthedocs.io/en/latest/
+  dev_url: https://github.com/toolsforexperiments/plottr
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - trevormorgan
```
Comment (astafan8) [inline recipes/plottr/meta.yaml:66]: shall we add wolfgang, Jens, and myself as well so that more people can work on this if needed?

**Comment by trevormorgan** [comment]
@conda-forge/help-python ready for review!

**Comment by trevormorgan** [comment]
@conda-forge/core ready for review

**Comment by trevormorgan** [comment]
@conda-forge/help-python ready for review

**Review by astafan8 — APPROVED** [review]

**Comment by jenshnielsen** [comment]
@conda-forge/help-python ready for review

**Pull request merged**